scala_211/lift_advanced_bs3/src/main/webapp/assets/css/app-bs-theme.css

```diff
@@ -56,9 +56,10 @@ body { padding-top: 70px; }
 }
 
 .navbar-default .navbar-nav > .open > a, 
+.navbar-default .navbar-nav > .active > a,
 .navbar-default .navbar-nav > .open > a:hover, 
 .navbar-default .navbar-nav > .open > a:focus {
-    background-color: #003C6F;
+    background-image: linear-gradient(to bottom, #003C6F 0%, #003C6F 100%);/*#003C6F;*/
```

Comment:
Usually you also specify a background color for non-gradient supporting browsers:

```css
  background-color: #003C6F;
  background-image: linear-gradient...;
```

Up to you if you want to do that here.